app/views/main.scala.html

```diff
@@ -7,6 +7,7 @@
         <title>@title</title>
         <meta name="description" content="">
         <meta name="viewport" content="width=device-width,maximum-scale=1"/>
+        <script src="@routes.Assets.versioned("javascripts/autotrack.js")"></script>
```

Comment:
If this is additional GA functionality, then I'd add `type="text/plain" class="optanon-category-2"` and place it below the OneTrust includes. This way it's only loaded if optanon-category-2 (Performance Cookies) are approved by the visitor.